### PR TITLE
feat(portal): include certificate name in update payloads and related…

### PR DESCRIPTION
**Pull request by aqibmohammadkhan**
… tests

Adds support for passing `activeCertificateName` in addition to `activeCertificateId` when updating certificates. Updates relevant tests to validate the changes.

## Issue

https://gravitee.atlassian.net/browse/APIM-13502

## Description

https://github.com/user-attachments/assets/a0d041f8-63d3-4910-82cd-bea724398109

### CODE REVIEW

**Review by dev-marek — APPROVED** [review]
If we weren't going for parity, this issue should be solved by creating a PATCH request/resource so that only expiry date would be updated. But in our situation this is an ok-ish solution.

File: gravitee-apim-portal-webui-next/src/app/dashboard/application-details/application-tab-settings/application-tab-settings-certificates/add-certificate-dialog/add-certificate-dialog.component.ts [hunk 1]
```diff
@@ -157,6 +158,7 @@ export class AddCertificateDialogComponent {
         switchMap(() => {
           if (gracePeriodEnd && this.data.activeCertificateId) {
             return this.certService.update(this.data.applicationId, this.data.activeCertificateId!, {
+              name: this.data.activeCertificateName,
```
[the remark this answers is not in the document]
Comment (aqibmohammadkhan) [reply gravitee-apim-portal-webui-next/src/app/dashboard/application-details/application-tab-settings/application-tab-settings-certificates/add-certificate-dialog/add-certificate-dialog.component.ts]: Also added a test case to make sure no empty name scenario.

**Comment by aqibmohammadkhan** [comment]
> If we weren't going for parity, this issue should be solved by creating a PATCH request/resource so that only expiry date would be updated. But in our situation this is an ok-ish solution.

I do agree with you but the current solution correctly addresses the bug by providing the mandatory `name` field in the `PUT` request to the certificate update endpoint. This approach maintains functional parity while ensuring data integrity during the certificate rotation process.
